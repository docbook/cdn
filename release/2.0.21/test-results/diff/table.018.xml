<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <p deltaxml:deltaV2="A">This test is a table with a long description.</p><div deltaxml:deltaV2="B" class="content">
            <p>This test is a table with a long description.</p>
            <p>CALS:</p>
            <figure id="R.1.4" class="table-wrapper">
               <div class="title">Table 1. A table with a title</div>
               <div class="longdesc-link">[<a class="dialog-link" href="#longdesc-xxx" title="long description link">D</a>]</div>
               <div class="table">
                  <table border="0" summary="Some short alt text" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                     <colgroup>
                        <col class="tcol1" width="25%"/>
                        <col class="tcol2" width="25%"/>
                        <col class="tcol3" width="25%"/>
                        <col class="tcol4" width="25%"/>
                     </colgroup>
                     <tbody>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a1</td>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="2" colspan="2">b1-c2</td>
                           <td style="border-bottom: 1px solid ; ">d1</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="3">a2-a4</td>
                           <td style="border-bottom: 1px solid ; ">d2</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b3</td>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">c3</td>
                           <td style="border-bottom: 1px solid ; ">d3</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b4</td>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; " colspan="2">c4-d4</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; ">a5</td>
                           <td style="border-right: 1px solid ; ">b5</td>
                           <td style="border-right: 1px solid ; "> </td>
                           <td>d5</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
            <p>HTML tables don't have long descriptions.</p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">CALS:</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.4" class="table-wrapper">
            <div class="title">Table 1. A table with a title</div>
            <div class="longdesc-link">[<a class="dialog-link" href="#longdesc-xxx" title="long description link">D</a>]</div>
            <div class="table">
               <table border="0" summary="Some short alt text" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                  <colgroup>
                     <col class="tcol1" width="25%"/>
                     <col class="tcol2" width="25%"/>
                     <col class="tcol3" width="25%"/>
                     <col class="tcol4" width="25%"/>
                  </colgroup>
                  <tbody>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a1</td>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="2" colspan="2">b1-c2</td>
                        <td style="border-bottom: 1px solid ; ">d1</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="3">a2-a4</td>
                        <td style="border-bottom: 1px solid ; ">d2</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b3</td>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">c3</td>
                        <td style="border-bottom: 1px solid ; ">d3</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b4</td>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; " colspan="2">c4-d4</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; ">a5</td>
                        <td style="border-right: 1px solid ; ">b5</td>
                        <td style="border-right: 1px solid ; "> </td>
                        <td>d5</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">HTML tables don't have long descriptions.</p>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>