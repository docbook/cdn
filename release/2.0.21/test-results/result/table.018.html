<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for table.018.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: table.018.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">7</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;This test is a table with a long description.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;CALS:&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;figure <span class="att">class="table-wrapper"</span> <span class="att">id="R.1.4"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="title"</span>&gt;Table 1. A table with a title&lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="longdesc-link"</span>&gt;[<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#longdesc-xxx"</span> <span class="att">title="long description link"</span>&gt;D&lt;/a&gt;</span>]&lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="table"</span>&gt;
               <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "</span> <span class="att">summary="Some short alt text"</span>&gt;
                  <span class="tag">&lt;colgroup&gt;
                     <span class="tag">&lt;col <span class="att">class="tcol1"</span> <span class="att">width="25%"</span>/&gt;</span>
                     <span class="tag">&lt;col <span class="att">class="tcol2"</span> <span class="att">width="25%"</span>/&gt;</span>
                     <span class="tag">&lt;col <span class="att">class="tcol3"</span> <span class="att">width="25%"</span>/&gt;</span>
                     <span class="tag">&lt;col <span class="att">class="tcol4"</span> <span class="att">width="25%"</span>/&gt;</span>
                  &lt;/colgroup&gt;</span>
                  <span class="tag">&lt;tbody&gt;
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;a1&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">colspan="2"</span> <span class="att">rowspan="2"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;b1-c2&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;d1&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">rowspan="3"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;a2-a4&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;d2&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;b3&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;c3&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;d3&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;b4&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">colspan="2"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;c4-d4&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; "</span>&gt;a5&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; "</span>&gt;b5&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; "</span>&gt; &lt;/td&gt;</span>
                        <span class="tag">&lt;td&gt;d5&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                  &lt;/tbody&gt;</span>
               &lt;/table&gt;</span>
            &lt;/div&gt;</span>
         &lt;/figure&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;HTML tables don't have long descriptions.&lt;/p&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;p&gt;This test is a table with a long description.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;CALS:&lt;/p&gt;</span>
            <span class="tag">&lt;figure <span class="att">class="table-wrapper"</span> <span class="att">id="R.1.4"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="title"</span>&gt;Table 1. A table with a title&lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="longdesc-link"</span>&gt;[<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#longdesc-xxx"</span> <span class="att">title="long description link"</span>&gt;D&lt;/a&gt;</span>]&lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="table"</span>&gt;
                  <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "</span> <span class="att">summary="Some short alt text"</span>&gt;
                     <span class="tag">&lt;colgroup&gt;
                        <span class="tag">&lt;col <span class="att">class="tcol1"</span> <span class="att">width="25%"</span>/&gt;</span>
                        <span class="tag">&lt;col <span class="att">class="tcol2"</span> <span class="att">width="25%"</span>/&gt;</span>
                        <span class="tag">&lt;col <span class="att">class="tcol3"</span> <span class="att">width="25%"</span>/&gt;</span>
                        <span class="tag">&lt;col <span class="att">class="tcol4"</span> <span class="att">width="25%"</span>/&gt;</span>
                     &lt;/colgroup&gt;</span>
                     <span class="tag">&lt;tbody&gt;
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;a1&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">colspan="2"</span> <span class="att">rowspan="2"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;b1-c2&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;d1&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">rowspan="3"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;a2-a4&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;d2&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;b3&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;c3&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;d3&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;b4&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">colspan="2"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;c4-d4&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; "</span>&gt;a5&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; "</span>&gt;b5&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">style="border-right: 1px solid ; "</span>&gt; &lt;/td&gt;</span>
                           <span class="tag">&lt;td&gt;d5&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                     &lt;/tbody&gt;</span>
                  &lt;/table&gt;</span>
               &lt;/div&gt;</span>
            &lt;/figure&gt;</span>
            <span class="tag">&lt;p&gt;HTML tables don't have long descriptions.&lt;/p&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This test is a table with a long description.</p><p>CALS:</p><figure id="R.1.4" class="table-wrapper"><div class="title">Table 1. A table with a title</div><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d43766e23" title="long description link">D</a>]</div><div class="table"><table border="0" summary="Some short alt text" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="25%" /><col class="tcol2" width="25%" /><col class="tcol3" width="25%" /><col class="tcol4" width="25%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a1</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="2" colspan="2">b1-c2</td><td style="border-bottom: 1px solid ; ">d1</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="3">a2-a4</td><td style="border-bottom: 1px solid ; ">d2</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b3</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">c3</td><td style="border-bottom: 1px solid ; ">d3</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b4</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " colspan="2">c4-d4</td></tr><tr><td style="border-right: 1px solid ; ">a5</td><td style="border-right: 1px solid ; ">b5</td><td style="border-right: 1px solid ; "> </td><td>d5</td></tr></tbody></table></div></figure><p>HTML tables don't have long descriptions.</p></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><p>This test is a table with a long description.</p><p>CALS:</p><figure id="R.1.4" class="table-wrapper"><div class="title">Table 1. A table with a title</div><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d86027e23" title="long description link">D</a>]</div><div class="table"><table border="0" summary="Some short alt text" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="25%" /><col class="tcol2" width="25%" /><col class="tcol3" width="25%" /><col class="tcol4" width="25%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a1</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="2" colspan="2">b1-c2</td><td style="border-bottom: 1px solid ; ">d1</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="3">a2-a4</td><td style="border-bottom: 1px solid ; ">d2</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b3</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">c3</td><td style="border-bottom: 1px solid ; ">d3</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b4</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " colspan="2">c4-d4</td></tr><tr><td style="border-right: 1px solid ; ">a5</td><td style="border-right: 1px solid ; ">b5</td><td style="border-right: 1px solid ; "> </td><td>d5</td></tr></tbody></table></div></figure><p>HTML tables don't have long descriptions.</p></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/table.018.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>