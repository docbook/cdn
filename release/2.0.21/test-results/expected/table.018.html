<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This test is a table with a long description.</p><p>CALS:</p><figure id="R.1.4" class="table-wrapper"><div class="title">Table 1. A table with a title</div><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d43766e23" title="long description link">D</a>]</div><div class="table"><table border="0" summary="Some short alt text" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="25%" /><col class="tcol2" width="25%" /><col class="tcol3" width="25%" /><col class="tcol4" width="25%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a1</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="2" colspan="2">b1-c2</td><td style="border-bottom: 1px solid ; ">d1</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; " rowspan="3">a2-a4</td><td style="border-bottom: 1px solid ; ">d2</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b3</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">c3</td><td style="border-bottom: 1px solid ; ">d3</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b4</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " colspan="2">c4-d4</td></tr><tr><td style="border-right: 1px solid ; ">a5</td><td style="border-right: 1px solid ; ">b5</td><td style="border-right: 1px solid ; "> </td><td>d5</td></tr></tbody></table></div></figure><p>HTML tables don't have long descriptions.</p></article><script src="../../resources/js/prism.js"></script></body></html>